src/update.ts

```diff
@@ -1,7 +1,35 @@
+import chalk from 'chalk'
 import * as updateNotifier from 'update-notifier'
 
 import * as pkg from '../package.json'
 
+const notifier = updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 })
+
 export default function notify() {
-  updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 }).notify({ isGlobal: true, isYarnGlobal: true })
+  if (notifier.update && notifier.update.latest !== pkg.version) {
+    const oldVersion = notifier.update.current
+    const latestVersion = notifier.update.latest
+    const changelog = `https://github.com/vtex/toolbelt/releases/tag/v${latestVersion}`
+    let type = notifier.update.type
+
+    switch (type) {
+      case 'major':
+        type = chalk.red(type)
+        break
+      case 'minor':
+        type = chalk.yellow(type)
+        break
+      case 'patch':
+        type = chalk.green(type)
+        break
+    }
+
+    notifier.notify({
+      isGlobal: true,
+      isYarnGlobal: true,
+      message: `New ${type} version of ${pkg.name} available! ${chalk.dim(oldVersion)} → ${chalk.green(latestVersion)}
+        ${chalk.yellow('Changelog:')} ${chalk.cyan(changelog)}
+        Run ${chalk.green(`yarn global add ${pkg.name}`)} to update!`,
```

Comment:
I think the manual line breaks here are messing a little with the table format:

![Screenshot from 2019-12-26 14-56-11](https://user-images.githubusercontent.com/26463288/71485479-f50a9580-27ef-11ea-8d5e-94fc7bd3f7b8.png)

If the message uses `/n`, like this:

```New ${type} version of ${pkg.name} available! ${chalk.dim(oldVersion)} → ${chalk.green(latestVersion)}\n${chalk.yellow('Changelog:')} ${chalk.cyan(changelog)}\nRun ${chalk.green(`yarn global add ${pkg.name}`)} to update!```

The message will be centralized:

![Screenshot from 2019-12-26 14-56-31](https://user-images.githubusercontent.com/26463288/71485530-3a2ec780-27f0-11ea-930f-2872ea32f30e.png)

Reply:
Nice catch! That was my 1st option but ESLint screamed at me because of Prettier.

We'd have to disable prettier for this line with `// eslint-disable-next-line prettier/prettier`, if that's not a problem.
<img width="1470" alt="Screen Shot 2019-12-26 at 3 08 12 PM" src="https://user-images.githubusercontent.com/14338588/71486022-effb1580-27f2-11ea-9f40-c42ca2afe44a.png">

Or we can use string concatenation instead of template literals, but I think it'd be against your style guide, wouldn't it?

Reply:
Oops, in that case try to run `yarn format` in the root of the project. This will run `prettier` to format and it will do it in a way that won't affect the string outcome.
You can too write the message by line, a way would be:
```js
notifier.notify({
      isYarnGlobal: true,
      isGlobal: true,
      message: [
        `New ${type} version of ${pkg.name} available! ${chalk.dim(oldVersion)} → ${chalk.green(latestVersion)}`,
        `${chalk.yellow('Changelog:')} ${chalk.cyan(changelog)}`,
        `Run ${chalk.green(`yarn global add ${pkg.name}`)} to update!`,
      ].join('\n'),
    })

```

Reply:
Prettier makes it a little strange, but you call the shots! What's best for you?
<img width="732" alt="Screen Shot 2019-12-26 at 4 13 34 PM" src="https://user-images.githubusercontent.com/14338588/71487539-b3331c80-27fa-11ea-9332-6c6a26e91c78.png">

Reply:
I think using the array will be prettier :)